WhaleSpotting/Models/Database/WhaleSighting.cs

```diff
@@ -10,10 +10,10 @@ public class WhaleSighting
     public DateTime DateOfSighting { get; set; }
     public float LocationLatitude { get; set; }
     public float LocationLongitude { get; set; }
-    public string PhotoImageURL { get; set; }
+    public string? PhotoImageURL { get; set; }
     public int NumberOfWhales { get; set; }
     public ApprovalStatus ApprovalStatus {get;set;}
-    public string Description { get; set; }
-    public WhaleSpecies WhaleSpecies { get; set; }
-    public User User { get; set; }
+    public string? Description { get; set; }
+    public WhaleSpecies? WhaleSpecies { get; set; }
+    public User? User { get; set; }
```

Comment:
Do we want these values to be nullable in the database?

Reply:
Nullable status changed